src/app/services/case/case.service.ts

```diff
@@ -14,14 +14,14 @@ const ADVANCED_SEARCH_CASE_PATH = '/api/cases/search';
   providedIn: 'root',
 })
 export class CaseService {
-  constructor(private readonly http: HttpClient, private ErrorHandler: ErrorHandlerService) {}
+  constructor(private readonly http: HttpClient, private ErrorHandlerService: ErrorHandlerService) {}
```

Comment:
Minor thing, probably should be camel case errorHandlerService: ErrorHandlerService otherwise it looks like a type or an enum and not a property of the class